cea/technologies/chiller_absorption.py

```diff
@@ -250,10 +220,11 @@ def calc_Cinv(qcold_W, locator, ACH_type, config):
 
                 InvC = Inv_a + Inv_b * (Q_nom_each_chiller) ** Inv_c + (Inv_d + Inv_e * Q_nom_each_chiller) * log(Q_nom_each_chiller)
                 Capex_a1 = InvC * (Inv_IR) * (1 + Inv_IR) ** Inv_LT / ((1 + Inv_IR) ** Inv_LT - 1)
-                Capex_a = Capex_a + Capex_a1
-                Opex_fixed = Opex_fixed + Capex_a1 * Inv_OM
+                Capex_a_ACH_USD = Capex_a_ACH_USD + Capex_a1
+                Opex_fixed_ACH_USD = Opex_fixed_ACH_USD + Capex_a1 * Inv_OM
+                Capex_ACH_USD = Capex_ACH_USD + InvC
 
-    return Capex_a, Opex_fixed
+    return Capex_a_ACH_USD, Opex_fixed_ACH_USD, Capex_ACH_USD
```

Comment:
This changes the return type of the function from a 2-tuple to a 3-tuple. That changes the functions signature. When doing this, make sure to update all consumers of this function! You can find them with ALT+F7

All other consumers of this function (see below) are now broken...

![image](https://user-images.githubusercontent.com/2969564/47796404-926d5500-dd24-11e8-928a-9ac0cb1991ac.png)